chardet/sbcharsetprober.py

```diff
@@ -118,4 +118,7 @@ def get_confidence(self):
             r = r * self._freq_char / self._total_char
             if r >= 1.0:
                 r = 0.99
+	    #print 'Charset name: %s confidence = %f' % (self._model['charset_name'], r)
```

Comment:
It was my mistake like with the my_test.py file. Unintentionally I've pulled to the github the working (temporary) directory instead of the final directory...